cmd/minishift/cmd/start.go

```diff
@@ -108,7 +109,12 @@ For the latter see 'minishift config -h'.`,
 		Run: runStart,
 	}
 
+	// enable experimental flags
+	//startCmd.Flags().BoolVar(&future, "future", false, "Enable experimental features (unsupported)")
```

Comment:
-1, I am not a fan of introducing some sort of future flag or env variable. I don't really see the benefit here. IMO it just gets more complicated.